tests/bpe.ini

```diff
@@ -51,15 +51,22 @@ preprocessors=[("source", "source_bpe", <bpe_preprocess>)]
 class=processors.bpe.BPEPreprocessor
 merge_file="tests/data/merges_100.bpe"
 
-
 [bpe_postprocess]
 class=processors.bpe.BPEPostprocessor
 
+;[bpe_vocabulary]
+;class=vocabulary.from_bpe
+;path="tests/data/merges_100.bpe"
 
+; The vocabulary.from_bpe method is incorrect.
+; Use vocabulary.from_dataset of vocabulary.from_file instead.
```

Comment:
to zahoď

Reply:
Ok, ale nekde bychom to meli mit poznacene, dokud tu metodu uplne nevyhodime.

Reply:
Tak rovnou udělej PR kterej do ní dá raise NotImplementedError s odkazem na tu issue na githubu